front/app/containers/IdeasShow/components/IdeaTitle.tsx

```diff
@@ -39,7 +39,7 @@ const IdeaTitle = ({ idea, projectId, translateButtonClicked }: Props) => {
   const ideaTitle = localize(idea.data.attributes.title_multiloc);
 
   return (
-    <IdeaHeader id="e2e-idea-title">
+    <IdeaHeader>
       <div aria-live="polite">
```

Comment:
We already use this ID in the `components/PostShowComponents/Title.tsx` component which is used in `IdeasShow`, so it wasn't necessary here.

Reply:
It would probably be clearest if we put it on `Container` of Title.tsx instead, instead of still defining it in two spots. :)